pkg/apis/sources/v1alpha1/test_helper.go

```diff
@@ -0,0 +1,63 @@
+/*
```

Comment:
This file seems identical to the one in v1alpha2/test_helper.go. Could we just use that one in this directory instead of duplicating the file?

Reply:
done